server/data/migrations/20190213212405_followups.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('followups', tbl => {
```

Comment:
Are we using "followup" for internal purposes and "refreshers" for front end?

Reply:
Good point, I was maybe on autopilot, but perhaps this should be renamed to refreshers

Reply:
Update this to refreshers and I'll go ahead and merge.